usr.sbin/bsdinstall/scripts/jail

```diff
@@ -52,11 +52,16 @@ error() {
 	fi
 }
 
-
 rm -rf $BSDINSTALL_TMPETC
 mkdir $BSDINSTALL_TMPETC
 mkdir -p $1 || error "mkdir failed for $1"
 
+if [ -n "$SCRIPT" ]
+then
+        split -a 2 -p '^#!.*' "$SCRIPT" $TMPDIR/bsdinstall-installscript-
+        . $TMPDIR/bsdinstall-installscript-aa
```

Comment:
What's the benefit of stripping the initial comments? #!... would be treated as a comment by /bin/sh. Can't you just .  the "$SCRIPT"?

Reply:
I didn't write the line, it comes from https://github.com/freebsd/freebsd-src/blob/main/usr.sbin/bsdinstall/scripts/script#L93

Reply:
and see ``SCRIPTING`` part : https://www.unix.com/man-page/freebsd/8/bsdinstall/

Reply:
The point is to split the 'script' file into two parts
everything before the sh-bang #! is variables to set, and everything after it is a 'post install' script to run at the end.

Reply:
yea, these scripts are 'special' and 'different' in ways that are opaque...  OK. this is fine then...